ip_range_check.py

```diff
@@ -1,9 +1,9 @@
-#!/usr/bin/python
+#!/usr/bin/python3
```

Comment:
If we hard-code this, it won't work on a default CentOS7 install, should we use the alternatives approach mentioned above to avoid that, even if we expect to deprecate CentOS7 and python2 support soon?

Reply:
The code would anyways not work with python2 (needs unicode)